CHANGES.md

```diff
@@ -99,7 +119,6 @@ To be released.
  -  `Swarm<T>` became not to fill states from trusted peers, because now states
     can be validated rather than trusted due to MPT.  [[#1117]]
  -  `HashDigest<SHA256>` became serializable.  [[#795], [#1125]]
-
```

Comment:
Is this change intended? Of course, it seems not to affect to the rendered result because it is markdown.

Reply:
No, it's removed by mistake.  I'm going to revert it.

Reply:
Removed.